src/pvsite_forecast.py

```diff
@@ -154,8 +154,13 @@ def convert_df(df: pd.DataFrame):
     csv = convert_df(df)
     now = datetime.now().isoformat()
 
+    MAEformula = abs(df['generation_power_kw'] - df['forecast_power_kw']).sum() / len(df)
+    MAE = round(MAEformula/1000,ndigits=3)
+    st.write("##### Current Mean Actual Error :", MAE, "MW")
```

Comment:
Could change the text to `f" Mean Absolute Error : {mae_rounded} MW"`
I find these f strings more readable